src/constants.js

```diff
@@ -121,6 +121,18 @@ export const itemStatusesTranslations = {
   [itemStatuses.RESTRICTED]: 'ui-requests.item.status.restricted',
 };
 
+export const requestTypeErrors = {
+  TITLE_LEVEL_ERROR: 'title',
+  ITEM_LEVEL_ERROR: 'item',
+};
+
+export const requestTypeErrorTranslations = {
```

Comment:
Could you please rename from requestTypeErrorTranslations to REQUEST_TYPE_ERROR_TRANSLATION_KEYS?

Reply:
done